pydatalab/pydatalab/routes/v0_1/admin.py

```diff
@@ -0,0 +1,103 @@
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+
+from pydatalab.config import CONFIG
+from pydatalab.mongo import flask_mongo
+from pydatalab.permissions import get_default_permissions
+
+admin = Blueprint("admins", __name__)
+
+
+@admin.before_request
```

Comment:
Looks great, can we add a test using the new `admin_client`/`client_admin` fixtures to check auth of these routes?